src/parquet/util/bit-stream-utils.inline.h

```diff
@@ -75,7 +75,7 @@ inline bool BitWriter::PutAligned(T val, int num_bytes) {
   return true;
 }
 
-inline bool BitWriter::PutVlqInt(int32_t v) {
+inline bool BitWriter::PutVlqInt(uint32_t v) {
```

Comment:
does this have any correctness implications? I assume that this function only ever was getting used with non-negative integers